flink-table-store-kafka/src/test/java/org/apache/flink/table/store/kafka/KafkaTableTestBase.java

```diff
@@ -92,7 +92,6 @@ protected void doStart() {
     @Before
     public void setup() {
         env = StreamExecutionEnvironment.getExecutionEnvironment();
-        tEnv = StreamTableEnvironment.create(env);
```

Comment:
Why remove this line?

Reply:
`KafkaTableTestBase` is a dedicated test base for preparing Kafka test container, it has nothing to do with a table environment, so I initialized `tEnv` in the subclass

Reply:
But it contains `StreamExecutionEnvironment` and `StreamTableEnvironment`, it has inited `StreamExecutionEnvironment`, why no `StreamTableEnvironment`?

Reply:
> But it contains `StreamExecutionEnvironment` and `StreamTableEnvironment`, it has inited `StreamExecutionEnvironment`, why no `StreamTableEnvironment`?

cause I want to create `tEnv` conditionally according to junit parameter . Like
```java
env.setRuntimeMode(executionMode);
if (executionMode == STRAMING) {
    env.enableCheckpointing(100);
}
tEnv = StreamTableEnvironment.create(env);
```

Reply:
You can just reinit it, It is not good to leave a null to a subclass